deployment/sklearn/sklearn-housing-regression-gbr.ipynb

```diff
@@ -0,0 +1,522 @@
+        "        }\n",
+        "        \n",
+        "    def example(self):\n",
+        "        return [[3.0, 1.0, 206.0, 110.0, 8.4, 1980.0, 1.0, 8801.0]]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "IjEeggm6lmgb"
+      },
+      "outputs": [],
+      "source": [
+        "artifacts = {'serialized_model': 'model.pkl'}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "NAcK09NzmddI",
+        "outputId": "192e6538-c11e-4bdb-d175-e342eb7b1a9e"
+      },
+      "outputs": [],
+      "source": [
+        "registered_model = client.get_or_create_registered_model(name='House Price Prediction')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "KiXnB4XCmSRW",
+        "outputId": "1bdd7c7b-0f5c-488a-d826-ce8309aad7ef"
+      },
+      "outputs": [],
+      "source": [
+        "model = registered_model.create_standard_model(\n",
+        "    name = 'gbr',\n",
+        "    model_cls = HousePricePredictor,\n",
+        "    environment = Python(requirements=['scikit-learn']),\n",
+        "    model_api = ModelAPI(X_train, y_train.to_frame(name='PricePrediction')),\n",
+        "    artifacts = artifacts\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "g5svsy82oNcW"
+      },
+      "source": [
+        "## 6. Model Endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "_IJAriGJm6Ip"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint('house-price-prediction-gbr')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ZSdMpp5ro_IZ",
+        "outputId": "ab60ce66-b4ad-429d-e73f-f9b08cca6e1e"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint.update(model, wait=True)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "fl30rJiXoYtn"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2V5vQ0NAj32d"
+      },
+      "source": [
+        "## 7. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "dqGPqltGo7Pd"
+      },
+      "outputs": [],
+      "source": [
+        "sample_data = X_train.values.tolist()[:5]"
```

Comment:
See my comment in the other notebook about `X_test`.

Reply:
Done.